README.md

```diff
@@ -1,9 +1,11 @@
 # filetree
-A FileTree for r/datacurator
+A filetree for r/datacurator
 
-Hello and Welcome to the [r/datacurator](https://reddit.com/r/datacurator) filetree repository.  
+Hello and welcome to the [r/datacurator](https://reddit.com/r/datacurator) filetree repository.  
 
-We aim to create a unified filetree for all kinds of data.
+We aim to create a unified filetree for all kinds of data, for making it easier to store, categorise and retrieve it.
```

Comment:
We aim to create a unified filetree for all kinds of data, which should help in storing, categorising and retrieving it.